src/app/public/modules/i18n/resources.pipe.ts

```diff
@@ -23,20 +31,34 @@ export class SkyAppResourcesPipe implements PipeTransform {
     private resourcesSvc: SkyAppResourcesService
   ) { }
 
+  // tslint:disable:max-line-length
   /**
    * Transforms a named resource string into its value.
-   * @param name The name of the resource string.
+   * @param args Either string (resource name) and any[] (template args) or SkyAppLocaleInfo, string (resource name) and any[] (template args).
    */
-  public transform(name: string, ...args: any[]): string {
```

Comment:
Let's hold off on modifying the pipe for now. If we find that we do need the pipe, we'll create a new one, perhaps `skyAppResourcesForLocale` where the first parameter would be the string name, the second the locale info, followed by params.

Reply:
Pipe changes removed.